.github/workflows/build_flang.yml

```diff
@@ -14,15 +14,15 @@ on:
 
 jobs:
   build_flang:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Hopefully it won't break anything. As we shifted to ubuntu-22.04, we're observing sudden segfault in `malloc()` in nag-tests...